src/controllers/ChooseAppealReasonController.ts

```diff
@@ -0,0 +1,41 @@
+import { SessionMiddleware } from 'ch-node-session-handler';
+import { controller } from 'inversify-express-utils';
+import { SafeNavigationBaseController } from './SafeNavigationBaseController';
+import { AppealReasonValidator } from './validators/AppealReasonValidator';
+
+import { AuthMiddleware } from 'app/middleware/AuthMiddleware';
+import { CompanyAuthMiddleware } from 'app/middleware/CompanyAuthMiddleware';
+import { createReasonsRadioGroup } from 'app/models/components/ReasonsRadioGroup';
+import { CHOOSE_REASON_PAGE_URI, OTHER_REASON_PAGE_URI, REVIEW_PENALTY_PAGE_URI } from 'app/utils/Paths';
+
+const template = 'choose-appeal-reason';
+
+const navigation = {
+    previous(): string {
+        return REVIEW_PENALTY_PAGE_URI;
+    },
+    next(): string {
+        return OTHER_REASON_PAGE_URI;
+    },
+    actions: (_: boolean) => {
+        return {
+            continue: 'action=continue'
+        };
+    }
+};
+
+@controller(CHOOSE_REASON_PAGE_URI, SessionMiddleware, AuthMiddleware, CompanyAuthMiddleware)
```

Comment:
Path handled by the controller should be not available if feature is disabled. We do not want only hide options visually, we want completely disable incomplete feature.

Reply:
Okay, I'll look into disabling the route. Would a simple redirect suffice or do I need to include logic to completely disable to controller when the reason isn't enabled?

Reply:
Resolved: Route disabling implemented by @n-r-b 's feature toggle middleware.